cmd/kube-proxy/app/server.go

```diff
@@ -776,6 +548,10 @@ func (s *ProxyServer) Run() error {
 	return nil
 }
 
+func (s *ProxyServer) birthCry() {
+	s.Recorder.Eventf(s.NodeRef, api.EventTypeNormal, "Starting", "Starting kube-proxy.")
+}
+
```

Comment:
Is this a golint recommendation?

Reply:
This is existing code. Am not sure I understand your question